ngReact.js

```diff
@@ -82,6 +113,31 @@
     }, {});
   }
 
+  /**
+   *
+   * @param watchDepth (value of HTML watch-depth attribute)
+   * @param scope (angular scope)
+   *
+   * Uses the watchDepth attribute to determine how to watch props on scope.
+   * If watchDepth attribute is NOT reference or collection, watchDepth defaults to deep watching by value
+   */
+  function watchProps (watchDepth, scope){
+    var args = Array.prototype.slice.call(arguments, 2);
+    var watchFn;
+
+    //default watchDepth to value if not reference or collection
+    if (isWatchCollectionAvailable && watchDepth === 'collection') {
```

Comment:
The isFeatureAvailable function would compare min for that feature against current angular version so min of 1.3.12 for a feature against 1.4 would return true.

However you're right, checking angular.isFunction(scope.$watchCollections) is alot simpler!  Thanks for pointing that out, I made the change in ngReact,js and minified the file